libs/driver/shopify/src/transforms/get-route.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Generates a route path based on the Shopify resource type and handle.
+ *
+ * @param type The Shopify resource type (COLLECTION, PRODUCT, PAGE, etc.)
+ * @param handle The resource handle/slug
+ * @returns The formatted route path
+ * @throws Error if type is empty
+ */
+export const getShopifyRoutePath = (type: string, handle: string) => {
+  if (!type) {
+    throw new Error('Type cannot be empty');
+  }
+
+  switch(type){
+    case 'COLLECTION':
+      return `/collections/${handle}`;
+    case 'PRODUCT':
+      return `/products/${handle}`;
```

Comment:
Iirc there is an enum for these prefixes, you should use that

Reply:
73d260407

Reply:
Oh I meant in the url, isn't there a enum value for the products prefix? I thought I saw that in a different PR